calypso/csadmin/clicontracts/lts_test.sh

```diff
@@ -14,9 +14,8 @@ testContractLTSInvoke() {
     eval $SED
     [ -z "$BC" ] && exit 1
 
-    OUTRES=`runCA contract lts spawn`
-    matchOK "$OUTRES" "^Spawned a new LTS contract. Its instance id is:
-[0-9a-f]{64}$"
+    testGrep "Spawned a new LTS contract. Its instance id is:
```

Comment:
@ineiti  By changing `matchOK` to `testGrep` you are actually changing the test. `matchOK` looks for an exact match, while `textGrep` only ensures that it finds AT LEAST the content....